graph/db/graph_test.go

```diff
@@ -3515,25 +3527,11 @@ func TestEdgePolicyMissingMaxHtcl(t *testing.T) {
 
 	// Attempting to deserialize these bytes should return an error.
 	r := bytes.NewReader(stripped)
-	err = kvdb.View(graph.db, func(tx kvdb.RTx) error {
-		nodes := tx.ReadBucket(nodeBucket)
-		if nodes == nil {
-			return ErrGraphNotFound
-		}
-
-		_, err = deserializeChanEdgePolicy(r)
-		if err != ErrEdgePolicyOptionalFieldNotFound {
-			t.Fatalf("expected "+
-				"ErrEdgePolicyOptionalFieldNotFound, got %v",
-				err)
-		}
-
-		return nil
-	}, func() {})
-	require.NoError(t, err, "error reading db")
+	_, err = deserializeChanEdgePolicy(r)
+	require.ErrorIs(t, err, ErrEdgePolicyOptionalFieldNotFound)
```

Comment:
Lol, so the whole `kvdb.View()` call was basically a no-op in the first place...

Reply:
indeed 🤓